earth_enterprise/rpms/buildSrc/src/main/groovy/GeeDeb.groovy

```diff
@@ -153,7 +159,16 @@ class GeeDeb extends com.netflix.gradle.plugins.deb.Deb {
             each {
                 requires(it)
             }
-
+        
+        if (autoSymLinkDetection) {
+            eachFile {
+                if (it.getFile().isFile() && Files.isSymbolicLink(it.getFile().toPath())) {
+                    link("/" + it.getRelativePath().toString(), Files.readSymbolicLink(it.getFile().toPath()).toString())
```

Comment:
I realize I may have suggested rewriting symlinks, but looking at here, I don't I'd want the symlink targets rewritten behind my back, at least, not without an explicit request for it.

I think, if this is to be fixed, it should be done in the `khEnvironment.installedExecutableSymlink` method.  That may be outside the scope of this ticket, so it may best to leave the link targets unmodified here.

Reply:
I was under the impression that the issue was that symlinks did not survive the copy operation done by Gradle. How would we go about fixing that issue in/with `khEnvironment.installedExecutableSymlink`?

Reply:
You're right.

I wasn't paying enough attention when reading this code.  It looks fine.